com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Network/NimbusClient.cs

```diff
@@ -31,6 +36,7 @@ public NimbusClient(CancellationTokenSource ctx, NimbusSDKConfiguration configur
 			Client.DefaultRequestHeaders.Add("Nimbus-Sdkv", platformSdkv);
 			Client.DefaultRequestHeaders.Add("Nimbus-Unity-Sdkv", "1.3.1");
```

Comment:
Feel free to start by moving it into a constant like the above strings. Automation is nice, but not required, manually updating the versions seems like an okay trade off to address @JasonSznol comment.